### PR TITLE
Add Bucket Logging Configuration support to the aws.s3.Bucket actor.

### PR DESCRIPTION
**Pull request by diranged**
Review @Siminm

### CODE REVIEW

**Comment by diranged** [comment]
@siminm :+1: ??

File: kingpin/constants.py [hunk 1]
```diff
@@ -46,3 +48,18 @@ class STATE(StringCompareBase):
     """
 
     valid = ('present', 'absent')
+
+
+class SchemaCompareBase(object):
```
Comment (diranged) [inline kingpin/constants.py]: I probably should document this, and the StringCOmpareBase in the developer.rst doc file..

Comment (siminm) [inline kingpin/constants.py]: What do you think about removing the check for `valid` parameter and always relying on `validate` method? The STATE would simply have `validate(self, option): return option in ('present', 'state')`

**Comment by siminm** [comment]
Left you a question. You can add the developer.rst docs in a separate PR. 👍

File: kingpin/constants.py (hunk 1 above)
Comment (diranged) [inline kingpin/constants.py]: I refactored the code a bit .. it can be continually improved, but lets focus on that outside of this PR.

**Pull request merged**